manifests/quick-start-mysql.yaml

```diff
@@ -555,7 +555,7 @@ spec:
         image: mysql:8
         name: main
         ports:
-        - containerPort: 5432
```

Comment:
that is very wrong! - but I think it must be changed in a different file - `mysql-deployment.yaml` and then run `make manifests`?

Reply:
Done, PTAL!